### PR TITLE
added EDA notebook

### PR DESCRIPTION
**Pull request by aphawkins1**

### CODE REVIEW

**Review by Andrewwango — CHANGES_REQUESTED** [review]
Love it. This PR should only bring in changes from ai002 ticket, not another ticket, and I don't want to re-review the changes to existing files from ai001 (README, requirements and data ingestion) Please remove these files from ai001 from this branch. Once ai001 has been merged, then merge main (the changes from ai001) into ai002 (you can do this with `git checkout ai002`, `git merge origin/main`) then reopen a PR.  

In terms of the EDA notebook, again rename it so it's clear it's part of the ai002 ticket. As before, also remove the dataset description from here as it's already in another ticket. 
Also include the code for creating the wordcloud. Finally, why aren't the EDA graphs/charts showing up? (may need a matplotlib inline magic command)

File: 002-EDA-notebook.ipynb [hunk 1]
```diff
@@ -0,0 +1,937 @@
...
...dWfRoEun/udyjc8YKFXHLk9mLl6Z/9h1R608zRXVXqhn9x0xc78YUX78CCY9HcniwQyrP6X5Pcxpz4BK+uMQd9AeICEy6bl0+Qca8vShFjTrhx5fXrKXgNeDCxDQSYRUH+HTphp2Cm5tc//lYnIcS3jBjs88QEL4QLhAFmkzGT4RS0IVTBC/GJdIXPPph8BT0njQFLzs4sXnyIIaqwMcjZ/OZch87n6Zodfbm0WI7HjKT/rDWuRZ4VZmOxDOkvdLDJ74FyzVR+Fo47ik/wdh8yTW1gem/20DhCGoNkYrNIkQibhAQvcpKT+pxJEbyO0MNsveOf3//8R72IhCp4cWRZ5kwZrxCzjrBAri1mwJzc8fiYYiYGM7k4y9W/4LzVao17qlQWpHoghrGpyn92E9djqXvpyo0qhvyFzqfbv1ebYZwTAoJxcnJig4ktG3GSmODtOug1lXfr/3LozFY5O9x1Cl5H6O348HWVB43ifF9DEbxIDylaurH0at9A5ef7FyeNAXGy5+8Dsn7haCXm4ysQ9liVCbwGKR4q1WNiT8meNbOKYcwa+4t49CVIv0hqX7x05SdqM2+wF3wbcZLcxhyc8Y0UqSUzBlxxVm44Yw5iB7P/jz1YRPnbv2CCBjPGiOXlqzeliDEnEuNYNN+TgteAdxLrfDD7gIEDO/QxM4pdvlgic0SpTsGLGTsMeDgWCCcZYNlt4fsfqU04oaY0YFMcZokhFDDAQBw6ZwnvP3hESj7XTlHFsS9PPhJ32dAftyPicGJAuZIPSYHr88off+1XS6RYgsesX9mSD6rZxkr1u6olZYh0CDCcOrDjm91qU0uTTsNl14+/q1k6bJZ5Y95KdQ9HrMUneB3x8MTDRdUObWzEwmw48h0xG+bkhhoIEWUisVMakvqcSRG8WN7D4I+XBmwAw489YGABs1AFLzhCbE0a0l7SprlKrsmT0/djIjj1A5sLETdOznd8nB0BhXx3LOPnzZVd+QlCBu1xhBw2nOFZq1cqKc9VKiFnz55Tm9aa1K2kji0rXb2D2vTU6sWqKmaRI4oYRDoF0gGCcUKbMPONnD5syMQPXSCvF0ugMTExcUSTiThJTPBu+3q31G89UH1vXqhRVvBSMO71hSptByc7oOgUvM5GQWyie+CeQrJz16+KK+InFMGL9iCf/MTJM9K9bT31XUc+JY6KctJfkJKAXO
```
[the remark this answers is not in the document]
Comment (Andrewwango) [reply 002-EDA-notebook.ipynb:907]: Sorry this doesn't seemed to have picked a nice bit of the diff to show - I'm referring to the last cell of markdown in the notebook

[the remark this answers is not in the document]
Comment (aphawkins1) [reply 002-EDA-notebook.ipynb:935]: Hi Andrew - I have added numerical values to the correlation matrix for sentence length and formality along with some comments :) I have also calculated the average word length to see how that correlates with formality, along with come comments.
Added 2 potential remedies to overcome the class imbalance :)

**Review by Andrewwango — APPROVED** [review]
All good, good luck with training

**Review by hellofadude — APPROVED** [review]

**Pull request merged**